tests/parsers/nexcelom_matrix/testdata/Cellaca_output_example.json

```diff
@@ -0,0 +1 @@
er (cell counter)": {"value": 13.02, "unit": "\u03bcm"}, "total cell count": {"value": 920, "unit": "cell"}, "viable cell count": {"value": 920, "unit": "cell"}}]}}]}}, {"measurement aggregate document": {"measurement document": [{"measurement time": "1970-01-01T00:00:00+00:00", "measurement identifier": "6a638594-de20-478c-a7f7-706585de7776", "device control aggregate document": {"device control document": [{"device type": "cell counter"}]}, "sample document": {"sample identifier": "P3-7"}, "processed data aggregate document": {"processed data document": [{"viability (cell counter)": {"value": 1.0, "unit": "%"}, "viable cell density (cell counter)": {"value": 1.8, "unit": "10^6 cells/mL"}, "total cell density (cell counter)": {"value": 1.8, "unit": "10^6 cells/mL"}, "average total cell diameter": {"value": 13.17, "unit": "\u03bcm"}, "average live cell diameter (cell counter)": {"value": 13.17, "unit": "\u03bcm"}, "total cell count": {"value": 915, "unit": "cell"}, "viable cell count": {"value": 915, "unit": "cell"}}]}}]}}, {"measurement aggregate document": {"measurement document": [{"measurement time": "1970-01-01T00:00:00+00:00", "measurement identifier": "5d9a0495-8138-41c3-83fd-6dd135b4f1b7", "device control aggregate document": {"device control document": [{"device type": "cell counter"}]}, "sample document": {"sample identifier": "P3-8"}, "processed data aggregate document": {"processed data document": [{"viability (cell counter)": {"value": 1.0, "unit": "%"}, "viable cell density (cell counter)": {"value": 1.73, "unit": "10^6 cells/mL"}, "total cell density (cell counter)": {"value": 1.73, "unit": "10^6 cells/mL"}, "average total cell diameter": {"value": 11.83, "unit": "\u03bcm"}, "average live cell diameter (cell counter)": {"value": 11.83, "unit": "\u03bcm"}, "total cell count": {"value": 882, "unit": "cell"}, "viable cell count": {"value": 882, "unit": "cell"}}]}}]}}, {"measurement aggregate document": {"measurement document": [{"measurement time": "1970-01-01T00:00:00+00:00", "measurement identifier": "2f76035b-bf14-4f77-b366-bad6fc674d72", "device control aggregate document": {"device control document": [{"device type": "cell counter"}]}, "sample document": {"sample identifier": "P3-9"}, "processed data aggregate document": {"processed data document": [{"viability (cell counter)": {"value": 1.0, "unit": "%"}, "viable cell density (cell counter)": {"value": 1.61, "unit": "10^6 cells/mL"}, "total cell density (cell counter)": {"value": 1.61, "unit": "10^6 cells/mL"}, "average total cell diameter": {"value": 10.74, "unit": "\u03bcm"}, "average live cell diameter (cell counter)": {"value": 10.74, "unit": "\u03bcm"}, "total cell count": {"value": 824, "unit": "cell"}, "viable cell count": {"value": 824, "unit": "cell"}}]}}]}}], "data system document": {"file name": "Cellaca output example.xlsx", "UNC path": "/Users/nick.collins/Downloads/Cellaca output example.xlsx", "ASM converter name": "allotropy", "ASM converter version": "0.1.30"}}}
```

Comment:
Looks like the `Cellaca` files are unused, should they be removed?

Reply:
@nathan-stender I've switched to the new(-ish?) testing pattern so it looks like this file is being used now